franklin/builder/models.py

```diff
@@ -47,12 +47,14 @@ def save(self, *args, **kwargs):
         base_path = os.environ['BASE_PROJECT_PATH']
         self.path = "{0}/{1}".format(base_path, self.repo_name)
         if not self.url:
+            repo_name = self.repo_name.split("/")[1]
             self.url = "{0}.{1}".format(
```

Comment:
nm. This is fine. By as discussed, we'll refactor how owner/name are split and stored.